packages/cli/src/commands/lint/types.ts

```diff
@@ -0,0 +1,77 @@
+type ConfigSidebarPage = {
```

Comment:
I think we should move the types, the schema and the code to check whether a given string matches the `ScalarConfig` schema to a separate `@scalar/config` package.

The CLI could then just import and execute the code, and we could eventually use the same types, validation and so on in docs.scalar.com.

Wdyt?

Reply:
Yes this is more organized 👍